src/librbd/Journal.cc

```diff
@@ -471,6 +624,31 @@ int Journal<I>::get_tag_owner(IoCtx& io_ctx, std::string& image_id,
 }
 
 template <typename I>
+void Journal<I>::is_tag_owner(I *image_ctx, ContextWQ *work_queue,
```

Comment:
@dillaman I can't find journal::OpenRequest/journal::CloseReques in master (only Create/Remove). May be they exist in some development branch? Also, I need to init only Journaler, though it might not make much difference.

Anyway, I think in future it could be optimized not to open/close journaler at all, just to get tags. I suppose a static method could be added to JournalMetadata...